### PR TITLE
Addon multiple reviews

### LINKED ISSUE #61: Add-on reviews to existing restaurant
As a User, I can add-on reviews to an existing restaurant so that I can add more food experiences to a particular food place.

### PR DESCRIPTION
**Pull request by Prabhakaran-Gokul**
Add multiple reviews to a specified entry in the Food Diary

### CODE REVIEW

File: src/main/java/fooddiary/logic/commands/AddOnCommand.java [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        model.updateFilteredEntryList(PREDICATE_SHOW_ALL_ENTRIES);
+        return new CommandResult(String.format(MESSAGE_ADDON_TO_ENTRY_SUCCESS, updatedEntry));
+    }
+
+    /**
+     * Creates and returns a {@code Entry} with the details of {@code entryToEdit}
+     * edited with {@code editPersonDescriptor}.
+     */
+    private static Entry createUpdatedEntry(Entry entryToAddOn, AddOnToEntryDescriptor addOnToEntryDescriptor) {
+        assert entryToAddOn != null;
```
Comment (marcusleeeugene) [inline src/main/java/fooddiary/logic/commands/AddOnCommand.java:84]: Good use of assert statement here!

File: src/main/java/fooddiary/model/entry/Review.java [hunk 2]
```diff
@@ -34,21 +39,40 @@ public static boolean isValidReview(String test) {
...
         return other == this // short circuit if same object
                 || (other instanceof Review // instanceof handles nulls
-                && value.equals(((Review) other).value)); // state check
+                && values.equals(((Review) other).values)); // state check
     }
 
     @Override
     public int hashCode() {
-        return value.hashCode();
+        return values.hashCode();
```
Comment (marcusleeeugene) [inline src/main/java/fooddiary/model/entry/Review.java:75]: Plural form here helps to identify the multiple values available, good job!

**Review by marcusleeeugene — APPROVED** [review]
Overall the code is good. Great job.

**Review by Donavanty — COMMENTED** [review]
LGTM, good job Gokul

**Review by Donavanty — APPROVED** [review]

**Pull request merged**